mitodl/bootcamp-ecommerce
# Allow alumni to skip application steps

#### What are the relevant tickets?
fixes #1166

#### What's this PR do?
Allow alumni to skip application steps 

#### How should this be manually tested?
Need to confirm that for the alumni it skips application steps and goes direc to `AWAITING_PAYMENT`

- run the migrations
- set `can_skip_application_steps` to `True` in the user profile
- set `allows_skip_steps` to `True` in the bootcamp run 
- apply for that specific bootcamp run

#### Where should the reviewer start?
(Optional)

#### Any background context you want to provide?
(Optional)

#### Screenshots (if appropriate)
**Alumni**

<img width="1440" alt="Screenshot 2021-03-31 at 14 45 00" src="https://user-images.githubusercontent.com/4043989/113148761-ae55e780-924b-11eb-96de-b043447298b5.png">
<img width="1440" alt="Screenshot 2021-03-31 at 14 45 55" src="https://user-images.githubusercontent.com/4043989/113148780-b31a9b80-924b-11eb-9caf-eabd6b80bae1.png">

**Non-Alumni**

<img width="1139" alt="Screenshot 2021-03-31 at 18 22 28" src="https://user-images.githubusercontent.com/4043989/113151016-21605d80-924e-11eb-8584-525ae0034a97.png">

**ADMIN PANEL**

<img width="1439" alt="Screenshot 2021-04-06 at 13 34 23" src="https://user-images.githubusercontent.com/4043989/113683058-35d4a800-96dd-11eb-9316-a7719175333a.png">
<img width="1440" alt="Screenshot 2021-04-06 at 13 34 57" src="https://user-images.githubusercontent.com/4043989/113683071-3a00c580-96dd-11eb-9b3c-3bf569432f5b.png">
